entries/d/deltadental.com.json

```
@@ -0,0 +1,51 @@
+{
+  "Delta Dental": {
+    "domain": "deltadental.com",
+    "img": "deltadental.com.png",
+    "tfa": [
+      "sms"
+    ],
+    "additional-domains": [
+      "deltadentalcoversme.com",
+      "deltadentalar.com",
+      "deltadentalks.com",
+      "deltadentalok.org",
+      "deltadentalpr.com",
+      "kentucky.deltadental.com",
+      "northcarolina.deltadental.com",
+      "tn.deltadental.com",
+      "www.deltadentalaz.com",
+      "www.deltadentalco.com",
+      "www.deltadentalct.com",
+      "www.deltadentalia.com",
+      "www.deltadentalid.com",
+      "www.deltadentalil.com",
+      "www.deltadentalin.com",
+      "www.deltadentalins.com",
+      "www.deltadentalma.com",
+      "www.deltadentalmi.com",
+      "www.deltadentalmn.org",
+      "www.deltadentalmo.com",
+      "www.deltadentalne.org",
+      "www.deltadentalnj.com",
+      "www.deltadentalnm.com",
+      "www.deltadentaloh.com",
+      "www.deltadentalri.com",
+      "www.deltadentalsc.com",
+      "www.deltadentalsd.com",
+      "www.deltadentalva.com",
+      "www.deltadentalwa.com",
+      "www.deltadentalwi.com",
+      "www.deltadentalwy.org",
+      "www.hawaiidentalservice.com",
+      "www.modahealth.com",
+      "www.nedelta.com"
```

We prefer if you don't list the subdomains unless it is needed to segregate services.
```suggestion
      "deltadentalaz.com",
      "deltadentalco.com",
      "deltadentalct.com",
      "deltadentalia.com",
      "deltadentalid.com",
      "deltadentalil.com",
      "deltadentalin.com",
      "deltadentalins.com",
      "deltadentalma.com",
      "deltadentalmi.com",
      "deltadentalmn.org",
      "deltadentalmo.com",
      "deltadentalne.org",
      "deltadentalnj.com",
      "deltadentalnm.com",
      "deltadentaloh.com",
      "deltadentalri.com",
      "deltadentalsc.com",
      "deltadentalsd.com",
      "deltadentalva.com",
      "deltadentalwa.com",
      "deltadentalwi.com",
      "deltadentalwy.org",
      "hawaiidentalservice.com",
      "modahealth.com",
      "nedelta.com"
```